src/sentry/processing/realtime_metrics/base.py

```diff
@@ -43,22 +93,42 @@ def get_lpq_projects(self) -> Set[int]:
         """
         pass
 
-    def add_project_to_lpq(self, project_id: int) -> None:
+    def add_project_to_lpq(self, project_id: int) -> bool:
         """
         Moves a project to the low priority queue.
 
         This forces all symbolication events triggered by the specified project to be redirected to
         the low priority queue, unless the project is manually excluded from the low priority queue
         via the `store.symbolicate-event-lpq-never` kill switch.
+
+        Returns True if the project was a new addition to the list. Returns False if it was already
+        assigned to the low priority queue.
```

Comment:
What does this return if the project wasn't in the lpq and hasn't been added because it's on the never list?